models/general/knowledge/en/loklak.txt

```diff
@@ -0,0 +1,11 @@
+# Return a list of universities in a country
+tweets about *| tweet *| 5 tweets about *|Latest tweets about *  
```

Comment:
I suggest you should add more questions to it to make this skill responsive like
Give me tweets about * 
Tell me tweets about *
what are latest tweets about * 
what are the latest tweets about * 
Can you give me tweets about * 
I want tweets about * 
I need tweets about * 

and similarly more like this.